daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/parsers/DelimiterParsers.scala

```diff
@@ -145,8 +147,8 @@ class DelimiterTextParser(
         localTypedDelims = localTypedDelims + " " + scannedDelims.next().lookingFor
       }
 
-      PE(start, "%s '%s' not found", delimiterType.toString, localTypedDelims)
-      return
+      if (!proceedOnMissingDelimiter)
+        PE(start, "%s '%s' not found", delimiterType.toString, localTypedDelims)
```

Comment:
Where do we require that the value was empty? It seems like this would allow a delimiter to be missing when there is a non-empty value?